### DISCUSSION TITLE
Interlisp Online Discussion

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#618] [Ideas]
@rmkaplan @stumbo @fghalasz   @skaisler1 @ecraven @johnwcowan
There's a lot to learn about how it all works and what the possibilities are. Here's a collection of notes from Monday and thoughts
* The Computer History Museum was interested in Medley running "in a browser" following the model of the Smalltalk Zoo. They're part of the "Software Preservation Network"., which is hosted by Educopia. EaaSi (Emulation as a Service infrastructure) is sponsored by SPN.  I think. But when we make UI choices as far as "who will use it", let's keep in mind that use case. @hjellinek Any more contact with Ethan@Yale?
* org stuff: there's an issue tag "online" ; use the interlisp/medley issue list.  I'm fiddling with Projects--ignore them for now.  
* keystroke need to be worked out; I wish I had a better model of what the requirements and current state -- maybe a google sheet?
* what goes where is an issue we've never resolved satisfactorily.  There are at least three separate contexts and audiences depending on MEDLEYDIR or lde installed or different package managers? Lispcore developer or novice user. set DWIMWAIT higher
* Execs Common Lisp vs. Interlisp or both? il:logout = xcl:logout?  Or a LOGOUT button? or different entry points to the container?
* It's actually an important thing to improve the experience of a tourist.

**Comment by hjellinek** [comment]
I haven't been in touch with Ethan@Yale in a while.  Our conversation centered on bibliographers, not EaaSI.
I'll fill you in on that on Monday - I ought to be able to attend the meeting.
